bayesian/utility/v2/ca_response_builder.py

```diff
@@ -433,6 +437,56 @@ def get_cve_maps(self) -> List[Dict]:
             ) for cve in self._cves]
         return cve_list
 
+    def get_premium_response(self) -> Dict:
+        """Get Premium Response.
+
+        :return: Dict of Registered User Response.
+        """
+        exploitable_vuls = self.get_exploitable_cves_counter()
+        logger.debug("Generating Premium Response.")
+        response = dict(
+            package=self.package,
+            version=self.version,
```

Comment:
The names of the fields in the premium response are different from the normal response, will this work in the UI?

Reply:
@prashbnair There is only 1 field name change

In Free Response, there is `registration_link` whereas in Premium Response we will get `vendor_package_link`
and
1 field is added in Premium Response, which was not a part of FreeTier Response: `exploitable_vulnerabilities_count`

That's what we settled with, in https://app.swaggerhub.com/apis/deepak1725/analytics-core_api_specs_for_version_2/2.1#/CA%20batch%20call/post_component_analyses

Reply:
Why do you have the package, version fields?
The component_analyses field is missing, severity is named highest_severity.

Reply:
> Why do you have the package, version fields?
@prashbnair This is Requested change by you to include version, package in each Package Recommendation.
I think we had a discussion regarding this in https://github.com/fabric8-analytics/fabric8-analytics-server/pull/655

>  The component_analyses field is missing, severity is named highest_severity.
Again this discussion happened in the mentioned PR. Please note I'm not unilaterally changing anything.